db/api/src/settings.rs

```diff
@@ -0,0 +1,230 @@
+use crate::{db::HoprDb, errors::Result};
+use async_trait::async_trait;
+use hopr_internal_types::prelude::AliasEntry;
+use sea_orm::{ActiveModelTrait, ColumnTrait, EntityTrait, QueryFilter};
+
+pub const ALIASES_ENTRY_NAME: &str = "aliases";
+
+/// Defines DB API for accessing HOPR settings (mainly aliases for now)
+#[async_trait]
+pub trait HoprDbSettingsOperations {
+    /// Retrieve the alias for a given PeerID
+    async fn get_alias(&self, alias: String) -> Result<Option<String>>;
+
+    /// Retrive all aliases
+    async fn get_aliases(&self) -> Result<Vec<AliasEntry>>;
```

Comment:
This API hates its user.

The alias to peer relationship suggests that the mapping is unique in this direction: `alias -> peer`. Why not use and pass back something like `HashMap<String, PeerId>`?

Reply:
If you enforce limitations on 1 alias for 1 peer id, even batter, because then you can use https://docs.rs/bimap/latest/bimap/

Reply:
And those are significantly simpler to use for the API user.